src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/ConfigurationLiveTests.cs

```diff
@@ -726,24 +751,85 @@ public async Task GetBatchSettingWithFields()
         {
             ConfigurationClient service = TestEnvironment.GetClient();
 
+            string key = string.Concat("keyFields-", Guid.NewGuid().ToString("N"));
+            string value = "my_value";
+            string label = "my_label";
+            ConfigurationSetting setting = await service.AddAsync(key, value, label);
+
+            string key2 = string.Concat("keyFields-", Guid.NewGuid().ToString("N"));
+            string value2 = "my_value_2";
```

Comment:
You never assert these values or use in any other way except calling AddAsync, does it matter to have them as variables?

Reply:
Not really. Removing